new version build docker has error!

Cargo.lock file content...
```
name = "clarinet-files"
version = "1.0.0"
  ```
**build error detail:**
```
   Compiling clarinet-files v1.0.1
   Compiling chainhook-sdk v0.5.0 (/src/components/chainhook-sdk)
error[E0599]: no variant or associated item named `Epoch22` found for enum `StacksEpochId` in the current scope
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/clarinet-files-1.0.1/src/project_manifest.rs:292:32
    |
292 |                 StacksEpochId::Epoch22
    |                                ^^^^^^^
    |                                |
    |                                variant or associated item not found in `StacksEpochId`
    |                                help: there is a variant with a similar name: `Epoch20`

error[E0599]: no variant or associated item named `Epoch23` found for enum `StacksEpochId` in the current scope
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/clarinet-files-1.0.1/src/project_manifest.rs:294:32
    |
294 |                 StacksEpochId::Epoch23
    |                                ^^^^^^^
    |                                |
    |                                variant or associated item not found in `StacksEpochId`
    |                                help: there is a variant with a similar name: `Epoch20`

error[E0599]: no variant or associated item named `Epoch24` found for enum `StacksEpochId` in the current scope
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/clarinet-files-1.0.1/src/project_manifest.rs:296:32
    |
296 |                 StacksEpochId::Epoch24
    |                                ^^^^^^^
    |                                |
    |                                variant or associated item not found in `StacksEpochId`
    |                                help: there is a variant with a similar name: `Epoch20`

error[E0599]: no variant or associated item named `Epoch22` found for enum `StacksEpochId` in the current scope
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/clarinet-files-1.0.1/src/project_manifest.rs:309:32
    |
309 |                 StacksEpochId::Epoch22
    |                                ^^^^^^^
    |                                |
    |                                variant or associated item not found in `StacksEpochId`
    |                                help: there is a variant with a similar name: `Epoch20`

error[E0599]: no variant or associated item named `Epoch23` found for enum `StacksEpochId` in the current scope
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/clarinet-files-1.0.1/src/project_manifest.rs:311:32
    |
311 |                 StacksEpochId::Epoch23
    |                                ^^^^^^^
    |                                |
    |                                variant or associated item not found in `StacksEpochId`
    |                                help: there is a variant with a similar name: `Epoch20`

error[E0599]: no variant or associated item named `Epoch24` found for enum `StacksEpochId` in the current scope
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/clarinet-files-1.0.1/src/project_manifest.rs:313:32
    |
313 |                 StacksEpochId::Epoch24
    |                                ^^^^^^^
    |                                |
    |                                variant or associated item not found in `StacksEpochId`
    |                                help: there is a variant with a similar name: `Epoch20`

For more information about this error, try `rustc --explain E0599`.
error: could not compile `clarinet-files` due to 6 previous errors
warning: build failed, waiting for other jobs to finish...

```

--- a/components/chainhook-cli/Cargo.toml
+++ b/components/chainhook-cli/Cargo.toml
@@ -6,8 +6,8 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-csv = "1"
-num_cpus = "1.4"
+csv = "1.2.2"
+num_cpus = "1.16.0"
 serde = "1"
 serde_json = "1"
 serde_derive = "1"
@@ -19,7 +19,7 @@ rand = "0.8.5"
 # raft-proto = { git = "https://github.com/tikv/raft-rs", rev="f73766712a538c2f6eb135b455297ad6c03fc58d", version = "0.7.0"}
 chainhook-sdk = { version = "0.5.0", default-features = false, features = ["ordinals", "zeromq"], path = "../chainhook-sdk" }
 chainhook-types = { version = "1.0.6", path = "../chainhook-types-rs" }
-clarinet-files = "1"
+clarinet-files = "1.0.1"
 hiro-system-kit = "0.1.0"
 # clarinet-files = { path = "../../../clarinet/components/clarinet-files" }
 # hiro-system-kit = { path = "../../../clarinet/components/hiro-system-kit" }
@@ -45,7 +45,8 @@ rocket = { version = "=0.5.0-rc.3", features = ["json"] }
 [dev-dependencies]
 criterion = "0.3"
 redis = "0.21.5"
-clarity-repl = "=1.5.0"
+clarity-repl = "=1.7.0"
+clarity-vm = "=2.1.1"
 hex = "0.4.3"
 
 

--- a/components/chainhook-sdk/Cargo.toml
+++ b/components/chainhook-sdk/Cargo.toml
@@ -13,7 +13,7 @@ serde_json = { version = "1", features = ["arbitrary_precision"] }
 serde-hex = "0.1.0"
 serde_derive = "1"
 stacks-rpc-client = "=1.0.7"
-clarinet-utils = "1"
+clarinet-utils = "1.0.0"
 hiro-system-kit = "0.1.0"
 # stacks-rpc-client = { version = "1", path = "../../../clarinet/components/stacks-rpc-client" }
 # clarinet-utils = { version = "1", path = "../../../clarinet/components/clarinet-utils" }

